node_scripts/submit.py

```diff
@@ -105,8 +105,10 @@ def __app_submit_cmd(
     spark_submit_cmd.add_option('--driver-class-path', driver_class_path)
     spark_submit_cmd.add_option('--driver-memory', driver_memory)
     spark_submit_cmd.add_option('--executor-memory', executor_memory)
-    spark_submit_cmd.add_option('--driver-cores', str(driver_cores))
-    spark_submit_cmd.add_option('--executor-cores', str(executor_cores))
+    if driver_cores:
```

Comment:
shouldn't we do this for all numerical settings (e.g. executor_memory)?

Reply:
I believe, executor memory is already a string (a numerical value and a unit) like "4g" or "1024M"